### ISSUE TITLE
pdf files are shrinked on deb and rpm packages

### ISSUE THREAD

**Issue by bruno-at-orange** [KhiopsML/khiops#2]
In the **deb** and **rpm** packages produced on github actions, the **pdf** files are shrinked:
```bash
 ls -gG usr/share/doc/khiops/
-rw-r--r-- 1   131 juin  19 16:51 KhiopsCoclusteringGuide.pdf
-rw-r--r-- 1   132 juin  19 16:51 KhiopsGuide.pdf
-rw-r--r-- 1   132 juin  19 16:51 KhiopsTutorial.pdf
```
When they are produced on my computer, we have 
```bash
ls -gG usr/share/doc/khiops/
-rw-r--r-- 1  854509 juin  19 09:58 KhiopsCoclusteringGuide.pdf
-rw-r--r-- 1 1736158 juin  19 09:58 KhiopsGuide.pdf
-rw-r--r-- 1 4115198 juin  19 09:58 KhiopsTutorial.pdf
```

**Comment by folmos-at-orange** [comment]
Are we still putting the guides in the packages? In Windows they are gone. In KhiopsML/OldKhiops#7 i'm eliminating these files from the repo.

**Comment by bruno-at-orange** [comment]
Ok, I thought it was only the samples that go out the packages. Pbm  fixed !

**Comment by folmos-at-orange** [comment]
So this is not anymore relevant. Is there any code in the linux packages that should be modified ? If no we can close this one

**Comment by folmos-at-orange** [comment]
I think you solved this in PR #26
